src/index.html

```diff
@@ -11,7 +11,194 @@
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            id="terms"
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="mail">E-mail:</label>
+          <input
+            id="mail"
+            placeholder="email@example.com"
+            type="email" name="email"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="pass">Password:</label>
+          <input
+            id="pass"
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="12"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+            Do you love cats?
+
+            <input
+              type="radio"
+              name="choosecat"
+              value="yes"
+            >
+              Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="choosecat"
+              value="no"
+            >
+              No
+          </label>
+
+        </div>
+
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              value="#000000"
+            >
+            </input>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+            >
+            </input>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select name="brands" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              name="rate"
+              value="0"
+              min="0"
+              max="10"
+              step="1"
+            >
+            </input>
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="forms">
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label>
+            Comments:
+            <textarea
+              name="comments"
+              id="comm"
+              cols="15"
+              rows="2"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
              name="comments"
              id="comm"
              cols="15"
              rows="2"
           ></textarea>
```
you may do like that 😉